Dockerfile.slim

```
@@ -0,0 +1,8 @@
+FROM alpine:3.11
+MAINTAINER sig-platform@spinnaker.io
+RUN apk --no-cache add --update bash openjdk11-jre
+RUN adduser -D -S spinnaker
```

```suggestion
RUN addgroup -S -g 10111 spinnaker
RUN adduser -D -S spinnaker -u 10111 -G spinnaker
```

We're going to add this to all the other files in the next week or two (because [a Kubernetes securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) requires a constant uid), so we might as well get ahead of things.